frontend/lib/ui/Checkbox/Checkbox.module.css

```diff
@@ -2,17 +2,55 @@
   display: inline-flex;
   align-items: center;
 
+  /* https://css-tricks.com/inclusively-hidden/ */
   input {
+    clip: rect(0 0 0 0);
+    clip-path: inset(50%);
+    height: 1px;
+    overflow: hidden;
+    position: absolute;
+    white-space: nowrap;
+    width: 1px;
+  }
+
+  > div {
+    display: inline-block;
     margin-right: 8px;
+    flex: 0 0 20px;
+    width: 20px;
+    height: 20px;
+    border-radius: 6px;
```

Comment:
Please convert the larger `px` amounts to `rem` (I made a lot of these changes in https://github.com/kiesraad/abacus/pull/303 for the codebase). See also `14px` on line 42.

Reply:
done